src/rubintv/templates/current.jinja

```diff
@@ -1,5 +1,5 @@
 {% extends "single_event.jinja" %}
 {% block footer_scripts %}
 <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
-<script type=text/javascript src="/rubintv/static/js/refresh-current.js"></script>
+<script type=text/javascript src="/rubintv/static/js/refresh-current.js?v=1234"></script>
```

Comment:
This doesn't affect present functionality, but I'd recommend a more automated approach to cache-busting, like a Jinja function or environment context that provides a dynamically calculated hash of the script file.